config/boards/rock-5a.wip

```diff
@@ -0,0 +1,28 @@
+# Rockchip RK3588S octa core 4/8/16GB RAM SoC eMMC USB3 USB-C GbE
+declare -g BOARD_NAME="Rock 5A"
+declare -g BOARDFAMILY="rockchip-rk3588"
+declare -g BOOTCONFIG="rock-5a-rk3588s_defconfig"
+declare -g BOOT_SOC="rk3588"
+declare -g KERNEL_TARGET="legacy"
+declare -g FULL_DESKTOP="yes"
+declare -g BOOT_LOGO="desktop"
+declare -g BOOT_FDT_FILE="rockchip/rk3588s-rock-5a.dtb"
+declare -g BOOT_SCENARIO="spl-blobs"
+declare -g WIREGUARD="no"
+declare -g IMAGE_PARTITION_TABLE="gpt"
+declare -g SKIP_BOOTSPLASH="yes" # Skip boot splash patch, conflicts with CONFIG_VT=yes
+declare -g BOOTFS_TYPE="fat"
+
+## only applies to extlinux SRC BOOT ARGS taken from radxa rock 5a image might need a fix
+declare -g SRC_EXTLINUX="yes" # going back to extlinux uboot for now
+declare -g SRC_CMDLINE="console=ttyFIQ0 console=tty1 consoleblank=0 loglevel=7 panic=10 rootwait rw init=/sbin/init rootfstype=ext4 cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory"
```

Comment:
Extlinux has less support on armbian, so better not define it.

Reply:
Dropped `rock-5a` from this PR, we should do it separately.